src/Middleware/NodeServices/samples/NodeServicesExamples/Controllers/HomeController.cs

```diff
@@ -1,10 +1,11 @@
+using System;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.NodeServices;
-using Microsoft.AspNetCore.SpaServices.Prerendering;
 
 namespace NodeServicesExamples.Controllers
 {
+    [Obsolete("Use Microsoft.AspNetCore.SpaServices.Extensions")]
```

Comment:
Do we need to obsolete the samples? Seems odd, since they're not shipped anyway.

Reply:
Same comment applies to other samples, but I'll not duplicate this everywhere.